test/api_integration/apis/saved_objects/find.ts

```diff
@@ -286,9 +286,8 @@ export default function ({ getService }: FtrProviderContext) {
               expect(resp.body).to.eql({
                 error: 'Bad Request',
                 message:
-                  'KQLSyntaxError: Expected AND, OR, end of input, ' +
-                  'whitespace but "<" found.\ndashboard.attributes.title:foo' +
-                  '<invalid\n------------------------------^: Bad Request',
+                  'KQLSyntaxError: Expected AND, OR, end of input but "<" found.\ndashboard.' +
+                  'attributes.title:foo<invalid\n------------------------------^: Bad Request',
```

Comment:
Since we're now shortcutting the `whitespace` rule when `parseCursor` is `false`, it's no longer included in the error messages as one of the acceptable alternatives (which should have been the case to begin with).